src/commands/misc/lorem.js

```diff
@@ -0,0 +1,34 @@
+const { SwitchbladeEmbed, Command, CommandError } = require('../../')
+const { LoremIpsum } = require('lorem-ipsum')
+
+module.exports = class Lorem extends Command {
+  constructor (client) {
+    super({
+      name: 'lorem',
+      parameters: [{
+        type: 'number',
+        full: true,
+        missingError: 'commands:lorem.noNumber',
+        required: false,
+        min: 1,
+        max: 10,
+      }]
+    },
+    client)
+
+    this.lorem = new LoremIpsum()
+  }
+
+  async run ({ channel, t }, paragraphs) {
+    if(paragraphs === undefined) {
+      paragraphs = 2
+    }
+
+    const loremString = this.lorem.generateParagraphs(paragraphs)
```

Comment:
```js
async run ({ channel, t }, paragraphs) {
  const loremString = this.lorem.generateParagraphs(paragraphs || 2)
```

Reply:
Should have thought about it! One-liner :)